consensus/consensus_service.go

```diff
@@ -346,6 +347,10 @@ func (consensus *Consensus) UpdateConsensusInformation() Mode {
 		}
 	}
 
+	if consensus.ChainReader.Config().IsFiveSeconds(nextEpoch) {
+		consensus.BlockPeriod = time.Duration(5 * time.Second)
```

Comment:
Shall we make 5s a const?

Reply:
This code is only temporary, after we enter that epoch, we can just remove this code and change the BlockPeriod flag to 5 seconds.

Reply:
Well, instead of providing the option to the user I would suggest to hard code this value in consensus. Because block time in flag is just a sanity field. However, according to protocol, it shall be enforced to be a certain value (8s or 5s). Parsing through flag would not be a great option for this "enforced" field as far as I see.

Reply:
we can just remove the flag then.

Reply:
It was there for testing purpose initially.

Reply:
> we can just remove the flag then.

Thanks, will add TODO for after my current cmd overhaul.